introduction.md

```diff
@@ -0,0 +1 @@
+Hello I am Nikolaos Leventakis, a Mechanical Engineer form Greece wishing to learn github. I a following codeschool.com git path.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.